mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3375,9 +3380,11 @@ async fn process_maker_connected(ctx: MmArc, from_pubkey: H256Json, connected: M
     log::debug!("Processing MakerConnected {:?}", connected);
     let ordermatch_ctx = OrdermatchContext::from_ctx(&ctx).unwrap();
 
-    let our_public_id = CryptoCtx::from_ctx(&ctx)
-        .expect("'CryptoCtx' must be initialized already")
-        .mm2_internal_public_id();
+    let our_public_id = match CryptoCtx::from_ctx(&ctx) {
+        Ok(ctx) => ctx.mm2_internal_public_id(),
+        Err(_) => return,
+    };
```

Comment:
I'm just curious, do you plan to use `process_maker_connected` in tests somewhere?

Reply:
I don't for now. Do you have any note on this if I would have such a plan? 🙂

Reply:
No, I asked to know why you decided to change) Have `process_maker_connected` or `process_taker_request` panicked in some case?

Reply:
I got `process_taker_request` panic report from WebDEX devs. `process_maker_connected` could panic as well because it doesn't have any prechecks.

Reply:
Oh I see, thanks! I didn't foresee it